clients/web/src/models/UserModel.js

```diff
@@ -1,6 +1,26 @@
 girder.models.UserModel = girder.Model.extend({
     resourceName: 'user',
 
+    // Sets this model to the currently logged in user; has no effect is no user
+    // is logged in.
+    current: function (success, error) {
```

Comment:
Whatever we decide on the [other question](https://github.com/girder/girder/pull/747#discussion_r27307636) will solve this as well.